src/python/CRABClient/Commands/status.py

```diff
@@ -57,7 +60,7 @@ def __call__(self):
         if not rootDagId:
             failureMsg = "The task has not been submitted to the Grid scheduler yet. Not printing job information."
```

Comment:
I would not call this a failure. In such cases DB status is NEW or QUEUED. Only if DB status is SUBMITTED and the rootDagId missing we can call it an error